.github/workflows/test.yml

```diff
@@ -1,12 +1,9 @@
 name: Build and Test JTReg
 
 on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - '*'
+  branches-ignore:
```

Comment:
Looks like you deleted one too many lines. You need "push:" before this line.

Reply:
Ooops, thanks for spotting that.